src/plugin/cesium/tiles/cesium3dtileslayer.js

```diff
@@ -65,10 +70,25 @@ plugin.cesium.tiles.Layer.prototype.synchronize = function() {
 
   if (!this.hasError()) {
     var tilesetUrl = '';
-    if (!isNaN(this.assetId)) {
+    if (!isNaN(this.assetId) && this.accessToken) {
       tilesetUrl = Cesium.IonResource.fromAssetId(this.assetId, {
         accessToken: this.accessToken
       });
+      tilesetUrl.then(() => {
+        console.log('Access token accepted');
```

Comment:
Two things:
- Avoid using `console` statements. All log messages should use `goog.log` at a level appropriate to the message.
- We don't really care if the promise is resolved, so the resolution parameter can be `undefined` here. We only care if it's rejected (invalid token).